test/vimish-fold-test.el

```diff
@@ -30,14 +30,16 @@
 (require 'cl-lib)
 (require 'vimish-fold)
 
-(defmacro with-test-file (&rest body)
-  "Run BODY in a temporary buffer with test file loaded."
+(defmacro with-test-file (file &rest body)
+  "Run `BODY' in a temporary buffer with test `FILE' loaded."
```

Comment:
Please remove quoting, see functions in defined in Emacs for examples of how function arguments should be formatted in the doc string.